src/SDKs/Search/DataPlane/Microsoft.Azure.Search/generate.cmd

```diff
@@ -0,0 +1,27 @@
+::
```

Comment:
@olydis Thanks! I'll make sure to sync up with you the next time we make a push on this, but it's probably not going to be until at least October.